src/plugins/colum-chooser/column-chooser.component.ts

```diff
@@ -0,0 +1,231 @@
+import {Component, Optional, OnInit, OnDestroy, Input, Output, EventEmitter, ChangeDetectorRef} from '@angular/core';
+import {Command} from 'ng2-qgrid/core/command';
+import {PipeUnit} from 'ng2-qgrid/core/pipe/pipe.unit2';
+import {GridService} from 'ng2-qgrid/main/grid';
+import * as columnService from 'ng2-qgrid/core/column/column.service';
+import {isFunction, noop} from 'ng2-qgrid/core/utility';
+import {Aggregation} from 'ng2-qgrid/core/services';
+
+import {RootService} from 'ng2-qgrid/infrastructure/component';
+import {PluginComponent} from '../plugin.component';
+import {TemplatePath} from 'ng2-qgrid/core/template';
+
+const GRID = 'qGrid';
+const COLUMN_CHOOSER_NAME = `${GRID}ColumnChooser`;
```

Comment:
remove this constants, make it in one style

Reply:
fixed: converted into single constant